### PR TITLE
3457: Allow multi-line poll options

### PR DESCRIPTION
**Pull request by Lakoja**
Fixes https://github.com/tuskyapp/Tusky/issues/3457

I'm not sure however that radio button or checkbox are top/start aligned with the text. I don't see an alignment at all for them. (and/or how does the text make room for them?)

### CODE REVIEW

File: app/src/main/res/layout/item_poll.xml [hunk 1]
```diff
@@ -11,8 +11,8 @@
         android:layout_height="wrap_content"
         android:layout_marginTop="6dp"
         android:background="@drawable/poll_option_background"
+        android:maxLines="3"
```
Comment (Lakoja) [inline app/src/main/res/layout/item_poll.xml:14]: There is also a lenght-limit on the poll option text (so we could get rid of this option). But maybe only for Mastodon?

**Comment by connyduck** [comment]
Can you show me a screenshot please?

**Comment by Lakoja** [comment]
> Can you show me a screenshot please?

grafik

**Review by connyduck — APPROVED** [review]
thx

File: app/src/main/res/layout/item_poll.xml (hunk 1 above)
Comment (connyduck) [reply app/src/main/res/layout/item_poll.xml:14]: I wouldn't overthink this

**Pull request merged**